docs/rules/use-collections-singleton-list.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Use Collections Singleton List
+description:
+    Replaces Arrays.asList with 0 or 1 parameters respectively with Collections.emptyList(...) or Collections.singletonList()
```

Comment:
Oops.. I think I meant Collections.emptyList() and Collections.singletonList(..). The dots are wrong.

Reply:
Fixed it.